content/1-introduction-2-review.md

```diff
@@ -0,0 +1,636 @@
+The resulting data is an approximation of the rapid evolution of artefacts in real life, yet in a controlled situation that can then be statistically analysed with access to all the parameters.
+This technique has been used in studies ranging from the evolution of visual patterns transmitted in a group of apes [@claidiere_cultural_2014], to the role of argumentation in the transmission of solutions to simple but counter-intuitive reasoning problems [@claidiere_argumentation_2017].
+\add{baumard-mutualistic-2013.}
+A number of other disciplines have developed methods that can be relevant to the study of CAT and SCE, and I review these in @sec:intro-review-neighbours.
+
+While authors have acknowledged that there is no core incompatibility between CAT and SCE [@acerbi_if_2015; @sterelny_cultural_2017] and a growing number of social scientists is discussing both approaches positively [see the comments to @mesoudi_towards_2006; and @slingerland_what_2008], the work I present in this thesis is mainly focused on CAT itself, and less so SCE.
+While the prevalence of the cultural attraction approach in the French academic landscape naturally had an impact on this choice, I see three other arguments for focusing on CAT.
+First, it puts cognitive science squarely in the middle of the problem because of the importance of psychology for the study culture.
+I admit to sharing that opinion (and given current evidence, it is not more than an opinion).
+Second, its initial goal is not so much to develop a mathematical theory of culture (which will surely flourish in due time), but to interest all disciplines studying human life in a common framework by using *principled philosophical arguments*.
+Third, this approach in turn leads to a clear articulation of the philosophical principles defended, discussed, and tested by the theory at the cultural and cognitive levels.^[
+  In the words of Tim Ingold, an outspoken critic of both CAT and SCE: "[Sperber's work] has the virtue of rendering unusually explicit the assumptions built into much contemporary theorising about culture and cognition, and of driving them through to their logical conclusions" [@ingold_transmission_2001, p. 113].
+]
+\add{smaldino-let-2014.}
+As such, it seems that CAT is in a good position to generate productive debate between alternative approaches (such as the propositions coming from social anthropology, and those discussed in @sec:intro-review-criticisms) which, at the moment, are still competing on the principled level.
+In particular, the challenges that arise when attempting to study CAT experimentally, as will be the case here for linguistic utterances, should prove helpful in developing a path to the resolution of the critiques faced by both CAT and SCE.
+
+
+### Neighbouring empirical areas {#sec:intro-review-neighbours}
+
+A number of adjacent disciplines have developed interest in and contributed to CAT.
```

Comment:
### Camille
"A number of adjacent disciplines", ce sont d'autres disciplines du coup?  je n'ai pas l'impression au vu des travaux cités.  Ou bien s'agit-il de "the theory has been applied in a number of adjacent cases, beyond cultural representations" ? (not sure either, but maybe closer)

Reply:
Done.